src/game.py

```diff
@@ -47,14 +47,37 @@ def get_user_input(self):
         user_input = input()
         return user_input
 
+    def is_winner(self, game_board):
+        if (
+            self.board.is_winner_horizontal(game_board)
+            or self.board.is_winner_vertical(game_board)
+            or self.board.is_winner_diagonal(game_board)
```

Comment:
should the game really know or care about these implementation details?  why not only have the game call `board.is_winner(game_board)`?

Reply:
alternatively, should the board know about the rules at all? I mean a game has rules and is played on a board. Are the rules a different concept all together?

Reply:
either way, I suggest not splitting the logic across classes.

Reply:
Hmm...now you got me thinking.  Does it make sense to have a Winner class??  That handles all the logic of winning...